packages/core/src/client.ts

```diff
@@ -1,5 +1,5 @@
 import { interpret } from 'xstate'
-
+import { NhostSession } from '.'
```

Comment:
I guess we are importing from the `index.ts` file here. ESLint would most likely complain about a circular dependency issue as the `index.ts` file also exports the client.

```suggestion
import { NhostSession } from './types'
```